backend/main/models/proposal.go

```diff
@@ -148,7 +148,8 @@ func (p *Proposal) IsLive() bool {
 // Returns an error if the account's balance is insufficient to cast
 // a vote on the proposal.
 func (p *Proposal) ValidateBalance(balance *Balance) error {
-	var weight uint64
+	var weight float64
```

Comment:
yes its true I left it like this so that it would compile. Otherwise I need to add some more code in the create endpoint and pass in a `VoteWithBalance`. But I remember in this case, no Votes would be created yet. Not sure how best we should handle that. update Balance to have `weight` ?

Reply:
yeah i think maybe just creating a VoteWithBalance struct so you can call the strategy's `GetVoteWeight` fn and pass that into this function would be ideal.

Reply:
cool!